topics/introduction/tutorials/galaxy-intro-short/slides.html

```diff
@@ -18,6 +18,16 @@
 contributors:
   - shiltemann
   - hexylena
+  - bagnacan
+  - bebatut
+  - pajanne
+  - nsoranzo
+  - chrisbarnettster
+  - michelemaroni89
+  - annefou
+  - nagoue
+  - olanag1
+  - tnabtaf
```

Comment:
These are relatively minor edits, shall we add just you here? (if we end up replacing the existing intro slide deck I am all for not erasing that attribution, but if we keep a long and a short version I would argue this one is different enough to be considered separately)